dynamics-nav/DATAFILEINFORMATION-Function--Database-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "DATAFILEINFORMATION Function (Database)"
+description: Describes the DATAFILEINFORMATION function (database) and provides syntax and parameters, 
```

Comment:
Describes the DATAFILEINFORMATION function (database) and provides syntax and parameters.